content/en/blog/trusted-boot.md

```diff
@@ -0,0 +1,155 @@
+
+
+## The Essence of Boot Security and Encryption
+
+![DALL·E](https://github.com/kairos-io/kairos-docs/assets/2420543/ea0a4d89-b037-44a2-a89b-7ad586787201)
+
+
+Here comes Trusted Boot and SENA to the rescue. Easier to explain with an Alice metaphor.
+
+> you can read more about Trusted Boot in https://0pointer.de/blog/brave-new-trusted-boot-world.html and about SENA here: https://kairos.io/blog/2023/04/18/kairos-is-now-part-of-the-secure-edge-native-architecture-by-spectro-cloud-and-intel/
+
+Let's imagine Alice has this super cool treasure chest, chock-full of her most prized stuff. She’s big on security, but in a high-tech way – kind of like how your computer needs to keep its data safe. To get this done, your computer uses some smart tricks, things like Trusted Boot, Full Disk Encryption (FDE), Secure Boot, and Measured Boot.
+
+To picture Full Disk Encryption, or FDE, think of it as Alice’s treasure chest having a lock that only opens with a secret code. It’s so clever that if someone breaks into her house and finds the chest, they still can’t get inside without that special code. FDE is like that for your computer – it scrambles all your data so that only someone with the right password can read it.
+
+Next, there's Secure Boot. Imagine if Alice had this smart doorbell. When someone presses it, the doorbell checks if they're on the guest list. If they're not, no entry. Secure Boot is your computer's version of this smart doorbell. It checks every piece of software when your computer starts up and only lets the good, trusted stuff in. It keeps out anything that looks fishy.
+And then, we have Measured Boot. Picture this as Alice keeping a super detailed logbook. She writes down everything that goes on in her house – who comes in, what they do, you name it. If anything odd pops up, she’s on it. Measured Boot does something similar for your computer. It watches over the startup process, takes notes, and makes sure everything's running just like it should, no funny business.
+
+Putting all these together, you’ve got Trusted Boot, which is like Alice's ultimate home security system – the secret-code lock, the smart doorbell, and the detailed logbook, all working together. We’re going to unpack how these nifty tools keep your computer as secure as Alice’s treasure chest, making sure your digital world is safe every time you power up.
+
+
+## Challenges and Solutions in Implementing FDE on Linux
+
+![DALL·E](https://github.com/kairos-io/kairos-docs/assets/2420543/aee89c19-a5d1-4b75-8b05-bbeef4e2b47e)
+
+
+While FDE is a standard feature in various operating systems, its implementation in Linux presents unique challenges. The initial boot stage, including the kernel and initial file system setup, cannot be fully encrypted, leaving a portion of the system exposed. However, through careful measures, this unencrypted segment can be protected against tampering, thus securing the system's boot process and the encrypted data it accesses.
```

Comment:
As a reader I would like to know why it isn't possible to fully encrypt. Doesn't need to be long but at least one phrase. Again, not critical, feel free to ignore

Reply:
good point! will address that in my next changeset